_posts/2017-03-17-implementing-the-schema-registry.md

```diff
@@ -0,0 +1,40 @@
+---
+layout:     post
+title:      Implementing The Schema Registry
+date:       2017-02-08 12:00
+summary:    Keeping on top of an area of technology that is as rapidly moving as the big data ecosystem is hard.  Our data tribe share some of their resources for keeping up to date.
+category:   Data
+tags:       Data, Improvements
+author:     callum_leahy
+---
+
+![Enigma Machine](/images/enigma.jpg)
+
+## Introduction
+
+The platform currently used within the Gaming Promotions Squad largely relies on Apache Kafka. Kafka is a data streaming service which is used to send data from one section of the stack to another. Apache Kafka works with Apache Avro, a data serialization system used to encode Kafka. As a result of using Avro we need to use a Schema Registry Service, this service will store the various Avro schemas that we require in order to encode and decode messages.
+
+## What it is
+
+The Confluent Schema Registry is an idea by the creators of the Kafka platform. It is a secure and safe way of storing schema versions and ensuring that an accurate history is kept in case a rollback is needed. The schema registry is stored in the _schema topic, which is on Kafka itself. So anyone who has access to Kafka will therefore have access to the schema registry. This gives us the GPS squad end-to-end schema availability meaning we can massively increase the reliability of a system.
```

Comment:
"This gives us the GPS squad end-to-end schema"
Should be (note the commas, I think it reads better that way):
"This gives us, the GPS squad, end-to-end schema"

Also, not sure about the use of 'GPS Squad' - that essentially says 'Gaming Promotions Squad Squad'.